doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-vscode-extension.adoc

```diff
@@ -18,3 +20,13 @@ After you install the VSCode extension bundle, any `.dmn`, `.bpmn`, or `.bpmn2`
 If the DMN, BPMN, or test scenario modelers open only the XML source of a DMN, BPMN, or test scenario file and displays an error message, review the reported errors and the model file to ensure that all elements are correctly defined.
 
 NOTE: For new DMN or BPMN models, you can also enter `dmn.new` or `bpmn.new` in a web browser to design your DMN or BPMN model in the online modeler. When you finish creating your model, you can click *Download* in the online modeler page to import your DMN or BPMN file into your {PRODUCT} project in VSCode.
+
+Before you can begin developing BPMN and DMN models in VSCode, you need to create a Maven project where you can build your assets and any other related resources for your application. To generate a Maven project using the default Maven archetype, navigate to a local folder and enter the following command in the command terminal:
```

Comment:
Recommend usage of VSCode extension is authoring of new business applications. (link to section with process)